service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/C21OrderEmailContentProviderTest.java

```diff
@@ -139,6 +157,24 @@ void shouldReturnExactC21NotificationParametersWithMostRecentUploadedDocumentUrl
                 "/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345");
     }
 
+    @Test
+    void shouldReturnExactC21LocalAuthorityNotificationParametersWithMostRecentUploadedDocumentUrl() {
```

Comment:
What is different in this test from the one declared above in line 115? Multi c21 are not relevant on that level as the URL is just provided in the contentProvider so why we need to test separate scenario? Is it even useful?

Reply:
yes looks very similar. Line 115 bit checks for Cafcass parameters and this checks for LA params